### ISSUE TITLE
Omit extra parent div

### ISSUE THREAD

**Issue by jschuler** [ankeetmaini/react-infinite-scroll-component#139]
Seems like that the component is rendering an extra div on the outside, e.g.
```
<div>
  <div class="infinite-scroll-component">...</div>
</div>
```
Is there any way to omit that outer div? Or at least add a class to it? It is causing issues in my CSS grid

**Comment by ankeetmaini** [comment]
This fix is not published yet. I'll do it soon! Thanks for reporting this.

**Comment by ForkBug** [comment]
Encounter the same issue. Will the npm be published in these days?

**Comment by ankeetmaini** [comment]
Really sorry @ForkBug, I just published 5.0! This is a major re-write in TS. Let me know if it works fine!

**Comment by ForkBug** [comment]
Awesome!!! You are doing so great job!
